### DISCUSSION TITLE
Problem to use INIT REQUEST a320

### DISCUSSION THREAD

**Discussion by guiguntherk10** [flybywiresim/aircraft#6932] [Help]
I've my A320 Pad connected with my SIMBRIEF USER, but as I see, I've tried to use the green button LINK MCDU, found that it's not working yet, then I've tried to load my SIMBRIEF plan and it's working, then I try to load using INIT REQUEST on my MCDU but it says "NO SIMBRIEF USER" have no idead about what's happening, may someone tell me how to fix it?

P.S btw I'm using the stable version

thanks a lot!

**Comment by DedaL54** [comment]
Are you logged in via flyped? (SimBrief, Navigraph USER NAME , **not** pilot ID! )
https://www.youtube.com/watch?v=xiMB6Jefk4o&list=PLoiGSEob7B3Uz-9uBBozGtfL2JGfc-Ow3&index=10&ab_channel=RealSimPilot
